skiko/src/commonMain/kotlin/org/jetbrains/skia/Data.kt

```diff
@@ -56,6 +56,13 @@ class Data internal constructor(ptr: NativePointer) : Managed(ptr, _FinalizerHol
             return Data(_nMakeUninitialized(length))
         }
 
+        fun makeFromFileName(path: String?): Data {
```

Comment:
Will it work for JS? If not, shouldn't it be de-commonized?

Reply:
Yep, it doesn't seem to make sense for JS, so API shape shall reflect that.

Reply:
I've reintroduced platform-specific makeFromFileName-s and corrected test accordingly